core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -458,8 +458,8 @@
       "The maximum number of files a compaction will open"),
   TSERV_COMPACTION_SERVICE_META_EXECUTORS(
       "tserver.compaction.major.service.meta.planner.opts.executors",
-      "[{'name':'small','maxSize':'32M','numThreads':2},"
-          + "{'name':'huge','numThreads':2}]".replaceAll("'", "\""),
+      ("[{'name':'small','maxSize':'32M','numThreads':2}," + "{'name':'huge','numThreads':2}]")
```

Comment:
If the string fits on a single line could remove the concatenation. Leaving the parens in case there is a bug w/ the formatter as you mentioned.

```suggestion
      ("[{'name':'small','maxSize':'32M','numThreads':2},{'name':'huge','numThreads':2}]")
```